### ISSUE TITLE
after update from 0.10 to 0.12 no login anymore

### ISSUE THREAD

**Issue by jorigel** [shaarli/Shaarli#1590]
Hi folks,
after update to 0.12 ( overwriting files and set rights ) there is no login possible anymore and the menu links don't work either. There comes up an :"The requested URL was not found on this server." error instead.

This is reproducible, what am I doing wrong?
ThanX for attention

jori

**Comment by ArthurHoaro** [comment]
Hi, it looks like you have a server configuration issue. What web server are you using?

Please also take a look at the documentation: https://shaarli.readthedocs.io/en/master/Server-configuration/

**Comment by jorigel** [comment]
Hi Arthur,
I'm using straight forward a sub-directory in apache2 on my own server inhouse. I was looking at the docu already, but wondering how this can change all over sudden. The old version was working fine. Is there anything different in the new version, that prevents from working in the same unchanged surrounding?

Thanks for trying to help me
regards
jori

**Comment by ArthurHoaro** [comment]
Shaarli now uses Slim framework, with URL rewriting. With Apache, it should work out of the box if you allow `.htaccess` override. I don't use Apache myself, but if the documentation does not help you, maybe this part is out of date.

**Comment by grov** [comment]
Hey guys, I had exactly the same issue. 
I forget to copy the .htaccess file in the root directory. Once the file was added, it worked fine. (don't miss the chown command too)

**Comment by jorigel** [comment]
Hi folks,
I tried all different kind of ways setting up a virtual host -> didnt work
using a subdirectory with a htaccess-file carrying the override -> didnt work either
allways the same result like described previously.

I'm not in the mood for a docker instance so I guess I stay to the old version and forget about the update - that works finally . . .
thanks for Your help guys
jori

**Comment by ArthurHoaro** [comment]
I'm sorry you weren't able to fix your issue. You might have a specific configuration preventing the redirections to be properly processed.

For the record, I just tried to follow the documentation step by step on a brand new EC2 instance, using @nodiscc screencast, and everything is working properly. Even in a subdirectory.

**Comment by jhautefeuille** [comment]
Same here with Openbsd 6.7...No Apache on this system but httpd with no .htaccess support.

**Comment by ArthurHoaro** [comment]
Hi,

Could you provide more information about your settings, such as your VHost configurations?

**Comment by jhautefeuille** [comment]
My vhost conf

```
server "shaarli.xyz.org" {
    include "/etc/httpd/httpd.listen.https.conf"
    root "/htdocs/shaarli"
    directory index index.php
    location "*.php*" {
        fastcgi socket "/run/php-fpm.sock"
    }
    include "/etc/httpd/httpd.auth.conf"
    include "/etc/httpd/httpd.log.conf"
    include "/etc/httpd/httpd.tls.conf"
    include "/etc/httpd/httpd.acme.conf"
}
```

**Comment by ArthurHoaro** [comment]
I didn't kwow OpenBSD's httpd, but it looks like it does not support URL rewriting - at least not easily. I have only found a few resources but [this](https://github.com/grocy/grocy/issues/623#issuecomment-603091928) and [this](http://karlbsd.blogspot.com/2016/05/c-amd64-x200-installation-for-drupal-8.html) (last paragraph).

I think I can make Shaarli work withtout URL rewriting by including `index.php` in the path.
Can you confirm that https://shaarli.xyz.org/index.php/picture-wall displays the picture wall page (styles and images won't load yet).

**Comment by jhautefeuille** [comment]
You can have more info about httpd daemon 

https://man.openbsd.org/httpd.conf.5

Yes I confirm `https://shaarli.xyz.org/index.php/picture-wall` displays wall page without styles and images.

Thank you

**Comment by ArthurHoaro** [comment]
I have merged #1593 which allows to use the latest version of Shaarli without URL rewriting (i.e. it loads style and images with URL prefixed by `/index.php`. It'll probably won't released immediately, so you can either use master version (it may contain not well tested new development) following the [documentation](https://shaarli.readthedocs.io/en/master/Server-configuration/#using-shaarli-without-url-rewriting), or rollback to the latest `v0.11.x` for the time being.

Note that if you manage to have a working configuration with OpenBSD's HTTP web server, we will gladly accept a pull request updating the server configuration [documentation](https://github.com/shaarli/Shaarli/blob/master/doc/md/Server-configuration.md).

**Comment by wakamex** [comment]
my default apache2 install on Ubuntu 22.04 LTS defaults to "Override None"
changing this from None to All in `/etc/apache2/apache2.conf` then restarting apache2 fixed it
```
<Directory /var/www/>
        Options Indexes FollowSymLinks
        AllowOverride **All**
        Require all granted
</Directory>
```
`sudo service apache2 restart`
